sites/all/modules/hr/hr_disasters/hr_disasters.features.field_base.inc

```diff
@@ -161,6 +161,36 @@ function hr_disasters_field_default_field_bases() {
     'type' => 'entityreference',
   );
 
+  // Exported field_base: 'field_hr_disaster_sync_disaster'
+  $field_bases['field_hr_disaster_sync_disaster'] = array(
```

Comment:
HR disasters looks like a feature in state Needs Review, you need to do a previous cleanup before creating a field, otherwise if we revert it, we might lose important bits of the feature